config.js

```diff
@@ -62,6 +62,11 @@ const ConfigOptions = () => {
         },
 
         /**
+         * Copy takes an object of assets to copy from src to dist folders
+         */
+        copy: {},
```

Comment:
I think it's because it's an empty object, if it had properties and you tried to override, then all the existing properties would be overwritten.

I think it's best to add it to the `update` — if only for consistencies sake?